messages/de/app.php

```diff
@@ -111,7 +111,7 @@
     'Redirecting' => 'Weiterleiten',
     'Reply' => 'Antworten',
     'Reply to Review' => 'Auf Rezension antworten',
-    'Report Description' => 'Beschreibung der Meldung',
+    'Report Description' => 'Beschreibung melden',
```

Comment:
This doesn't mean reporting a description, but the description of the report. The original translation was correct.

Reply:
Oh ok sry